taxi/views.py

```diff
@@ -1,9 +1,11 @@
 from django.contrib.auth.decorators import login_required
 from django.shortcuts import render
+from django.urls import reverse_lazy
 from django.views import generic
 from django.contrib.auth.mixins import LoginRequiredMixin
+from taxi.forms import CarForm, ManufacturerForm
 
-from .models import Driver, Car, Manufacturer
+from taxi.models import Driver, Car, Manufacturer
```

Comment:
Separate different groups of imports correctly with blank line according to PEP-8